server/build.gradle

```diff
@@ -304,17 +304,22 @@ thirdPartyAudit.excludes = [
   'com.google.common.geometry.S2LatLng',
 ]
 
-if (JavaVersion.current() <= JavaVersion.VERSION_1_8) {
-  // Used by Log4J 2.11.1
+if (project.runtimeJavaVersion <= JavaVersion.VERSION_1_8) {
   thirdPartyAudit.excludes += [
-    'java.io.ObjectInputFilter',
-    'java.io.ObjectInputFilter$Config',
-    'java.io.ObjectInputFilter$FilterInfo',
-    'java.io.ObjectInputFilter$Status'
+          // Used by Log4J 2.11.1
+            'java.io.ObjectInputFilter',
+            'java.io.ObjectInputFilter$Config',
+            'java.io.ObjectInputFilter$FilterInfo',
+            'java.io.ObjectInputFilter$Status',
+          // added in 9
+            'java.lang.ProcessHandle',
```

Comment:
AFAIK this came up now as it's not in 8 for sure. 
Maybe Gradle had something on the cp for compatibility ?